subprojects/ide-native/src/main/java/org/gradle/ide/visualstudio/VisualStudioExtension.java

```diff
@@ -20,17 +20,12 @@
 import org.gradle.api.NamedDomainObjectSet;
 
 /**
- * The configuration for mapping a set of {@link org.gradle.nativeplatform.NativeComponentSpec}s to a Visual Studio project.
+ * The configuration for mapping a set of C++ components to a Visual Studio project.
```

Comment:
Maybe just 'native components'?  It's more than C++

Reply:
Heh, I honestly have no recollection of making this change.  But you're right - I changed it here and in a few other places, too.